src/cpp/processor/Processor.cpp

```diff
@@ -517,6 +527,41 @@ bool Processor::process_reset_submessage(ProxyClient& client, InputPacket& /*inp
     return true;
 }
 
+#ifdef PERFORMANCE_TESTING
+bool Processor::process_performance_submessage(ProxyClient& client, InputPacket& input_packet)
+{
+    /* Set output packet. */
+    OutputPacket output_packet;
+    output_packet.destination = input_packet.source;
+
+    /* Get epoch time and array. */
+    uint8_t buf[65536];
```

Comment:
Use: `std::numeric_limits<uint16_t>::max()` instead? this function is a `constexpr`.

Reply:
I prefer to use `UINT16_MAX` macro.